tests/system/providers/amazon/aws/example_dynamodb_to_s3.py

```diff
@@ -134,7 +134,8 @@ def delete_dynamodb_table(table_name: str):
         dynamodb_table_name=table_name,
         file_size=1000,
         s3_bucket_name=bucket_name,
-        export_time=datetime(year=2023, month=4, day=10),
+        export_time=datetime.now(),
+        s3_key_prefix=f"{S3_KEY_PREFIX}-3-",
```

Comment:
Why are we hard-coding the `-3-` part?

Reply:
No reason was just following other examples in the file.

Reply:
I can get behind that reasoning.  I didn't see the other two in the github "changes" diff, so I didn't catch the pattern.